optimade/server/config.py

```diff
@@ -1,260 +1,68 @@
 import os
 import json
-from typing import Any
+from typing import Any, Optional, Dict, List
+from typing_extensions import Literal
```

Comment:
I think this is added by standard into `typing` from Python 3.8+? It's great to see it here as well, as if we're using `__future__` or something 😅

Reply:
I believe you're correct. It's needed for the 3.6 and 3.7. It doesn't hurt in 3.8. I can modify it so it's conditional import on `typing` first and if that fails, switch to `typing_extensions`.